File: 242-ValidAnagram/242-ValidAnagram.java
// Last updated: 8/1/2025, 10:38:08 PM
import java.util.*;
class Solution {
    public boolean isAnagram(String s, String t) {
       
        char[] charArray1 = s.toCharArray();
Arrays.sort(charArray1);
String sortedString1 = new String(charArray1);
char[] charArray2 = t.toCharArray();
Arrays.sort(charArray2);
String sortedString2 = new String(charArray2);
return sortedString2.equals(sortedString1);
    }
}

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 8/1/2025, 10:38:15 PM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
      
       if len(s)!=len(t):
         return False
       k=sorted(s)
       m=sorted(t)
       
       k = sorted(s.replace(" ", ""))
       m = sorted(t.replace(" ", ""))

       return k==m
          